<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>501</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	background-color: rgb(197 , 195,196);
	border-radius: 0px;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>741</width>
      <height>451</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(197 , 195,196);

	border-radius: 15px;
border: 1px solid rgb(0,0,0);
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="software_settings">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(100,100,100);

	color: rgb(0,0,0);
	border: none;
	text-align: center;
	padding-left: 10px;
	border-radius: 15px;
	border-left: 1px solid rgb(30, 30, 30);
	border-right:1px solid rgb(30, 30, 30);
}

QPushButton:hover {
border: 1px solid rgb(0,0, 0);
}

QPushButton:pressed {
	background-color: rgb(0, 0, 0);
}

</string>
     </property>
     <property name="text">
      <string> General Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="camera_settings">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>173</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(100,100,100);

	color: rgb(0,0,0);
	border: none;
	text-align: center;
	padding-left: 10px;
	border-radius: 15px;
	border-left: 1px solid rgb(30, 30, 30);
	border-right:1px solid rgb(30, 30, 30);
}

QPushButton:hover {
border: 1px solid rgb(0,0, 0);
}

QPushButton:pressed {
	background-color: rgb(0, 0, 0);
}

</string>
     </property>
     <property name="text">
      <string>   Camera Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="manage_user_btn">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>320</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(100,100,100);

	color: rgb(0,0,0);
	border: none;
	text-align: center;
	padding-left: 10px;
	border-radius: 15px;
	border-left: 1px solid rgb(30, 30, 30);
	border-right:1px solid rgb(30, 30, 30);
}

QPushButton:hover {
border: 1px solid rgb(0,0, 0);
}

QPushButton:pressed {
	background-color: rgb(0, 0, 0);
}

</string>
     </property>
     <property name="text">
      <string>User Setting</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pop_settings">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>321</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(100,100,100);
	color: rgb(0,0,0);
	border: none;
	text-align: center;
	padding-left: 10px;
	border-radius: 15px;
	border-left: 1px solid rgb(30, 30, 30);
	border-right:1px solid rgb(30, 30, 30);
}

QPushButton:hover {
border: 1px solid rgb(0,0, 0);
}

QPushButton:pressed {
	background-color: rgb(0, 0, 0);
}

</string>
     </property>
     <property name="text">
      <string>Communication Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="report_settings">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>173</y>
       <width>321</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(100,100,100);

	color: rgb(0,0,0);
	border: none;
	text-align: center;
	padding-left: 10px;
	border-radius: 15px;
	border-left: 1px solid rgb(30, 30, 30);
	border-right:1px solid rgb(30, 30, 30);
}

QPushButton:hover {
border: 1px solid rgb(0,0, 0);
}

QPushButton:pressed {
	background-color: rgb(0, 0, 0);
}

</string>
     </property>
     <property name="text">
      <string>Report Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="calibration_settings">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>321</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(100,100,100);

	color: rgb(0,0,0);
	border: none;
	text-align: center;
	padding-left: 10px;
	border-radius: 15px;
	border-left: 1px solid rgb(30, 30, 30);
	border-right:1px solid rgb(30, 30, 30);
}

QPushButton:hover {
border: 1px solid rgb(0,0, 0);
}

QPushButton:pressed {
	background-color: rgb(0, 0, 0);
}

</string>
     </property>
     <property name="text">
      <string>Calibration</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="headerFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100,100,100);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/milad/Desktop/dorsa2.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="margin">
        <number>-1</number>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>370</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>326</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QFrame" name="userFrame">
       <property name="minimumSize">
        <size>
         <width>356</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame {
	background-color: rgb(100,100,100);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="login_welcome">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="user_btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>26</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border: none;
	background-color: rgb(100,100,100);
	border-left: 1px solid rgb(30, 30, 30);
	border-right:1px solid rgb(30, 30, 30);
	color: rgb(255, 85, 0);
	border-radius: 13px;
}

QPushButton:hover {
border: 1px solid rgb(0,0, 0);
}

QPushButton:pressed {
	background-color: rgb(25, 25, 25);
}
</string>
          </property>
          <property name="text">
           <string>Guest</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/icons/icons8_account_1.svg</normaloff>:/icons/icons/icons8_account_1.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toggleButton_6">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border: none;
	background-color: rgb(100,100,100);
	border-left: 1px solid rgb(30, 30, 30);
	border-right:1px solid rgb(30, 30, 30);
	color: rgb(255, 85, 0);
	border-radius: 13px;
}

QPushButton:hover {
border: 1px solid rgb(0,0, 0);
}

QPushButton:pressed {
	background-color: rgb(25, 25, 25);
}
</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/icons/icons8_alarm.svg</normaloff>:/icons/icons/icons8_alarm.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_btn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: none;
	background-color: rgb(100,100,100);
}

QPushButton:hover {
	background-color: rgb(255, 38, 0);
}

QPushButton:pressed {
	background-color: rgb(255, 0, 0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/icons8_multiply.svg</normaloff>:/icons/icons/icons8_multiply.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
